### PR TITLE
Fixes #369

### LINKED ISSUE #369: Signing Jitsi Meet
Hello! I bought an Apple Developer account and I have the developer certificates installed on my computer.

I am having trouble running 'npm run dist' on Jitsi Meet. Electron-builder automatically detected my certificates and started to sign it. However.... this occurred: 
```
 • packaging       platform=darwin arch=x64 electron=9.0.0 appOutDir=dist/mac
  • downloading     url=https://github.com/electron/electron/releases/download/v9.0.0/electron-v9.0.0-darwin-x64.zip size=73 MB parts=8
  • downloaded      url=https://github.com/electron/electron/releases/download/v9.0.0/electron-v9.0.0-darwin-x64.zip duration=35.845s
  • signing         file=dist/mac/Jitsi Meet.app identityName=Developer ID Application: Madison Wang (4MP829ALF6) identityHash=AB3B90EFED7B0947C2CD305DE9AC1C0098F2B50D provisioningProfile=none
  ⨯ Command failed: codesign --sign AB3B90EFED7B0947C2CD305DE9AC1C0098F2B50D --force --options runtime --entitlements entitlements.mac.plist /Users/mac/Documents/jitsi-meet-electron/dist/mac/Jitsi Meet.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework
/Users/mac/Documents/jitsi-meet-electron/dist/mac/Jitsi Meet.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework: code object is not signed at all
In subcomponent: /Users/mac/Documents/jitsi-meet-electron/dist/mac/Jitsi Meet.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Helpers/chrome_crashpad_handler
  stackTrace=
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 Error: Command failed: codesign --sign AB3B90EFED7B0947C2CD305DE9AC1C0098F2B50D --force --options runtime --entitlements entitlements.mac.plist /Users/mac/Documents/jitsi-meet-electron/dist/mac/Jitsi Meet.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 /Users/mac/Documents/jitsi-meet-electron/dist/mac/Jitsi Meet.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Electron Framework: code object is not signed at all
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 In subcomponent: /Users/mac/Documents/jitsi-meet-electron/dist/mac/Jitsi Meet.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Helpers/chrome_crashpad_handler
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     at ChildProcess.exithandler (child_process.js:295:12)
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     at ChildProcess.emit (events.js:210:5)
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     at maybeClose (internal/child_process.js:1021:16)
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 From previous event:
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     at module.exports.execFileAsync (/Users/mac/Documents/jitsi-meet-electron/node_modules/app-builder-lib/electron-osx-sign/util.js:41:10)
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     at /Users/mac/Documents/jitsi-meet-electron/node_modules/app-builder-lib/electron-osx-sign/sign.js:172:17
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 From previous event:
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     at signApplicationAsync (/Users/mac/Documents/jitsi-meet-electron/node_modules/app-builder-lib/electron-osx-sign/sign.js:127:6)
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     at /Users/mac/Documents/jitsi-meet-electron/node_modules/app-builder-lib/electron-osx-sign/sign.js:338:14
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     at processTicksAndRejections (internal/process/task_queues.js:93:5)
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     at MacPackager.sign (/Users/mac/Documents/jitsi-meet-electron/node_modules/app-builder-lib/src/macPackager.ts:202:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! jitsi-meet-electron@2.1.1 dist: `npm run build && electron-builder`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the jitsi-meet-electron@2.1.1 dist script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```

### PR DESCRIPTION
**Pull request by awlx**
It adds a new electron-builder version to fix deep signing. It's tested with FreifunkMeet 2.0.14 and works.

Fixes #369

```

  • packaging       platform=darwin arch=x64 electron=9.0.1 appOutDir=dist/mac
  • downloading     url=https://github.com/electron/electron/releases/download/v9.0.1/electron-v9.0.1-darwin-x64.zip size=73 MB parts=8
  • downloaded      url=https://github.com/electron/electron/releases/download/v9.0.1/electron-v9.0.1-darwin-x64.zip duration=13.518s
  • signing         file=dist/mac/FreifunkMeet.app identityName=Developer ID Application:  identityHash=provisioningProfile=none
  • building        target=macOS zip arch=x64 file=dist/ffmuc-meet.zip
  • building        target=DMG arch=x64 file=dist/ffmuc-meet.dmg
  • building block map  blockMapFile=dist/ffmuc-meet.dmg.blockmap
  • building embedded block map  file=dist/ffmuc-meet.zip
```

### CODE REVIEW

**Comment by saghul** [comment]
Thank you!

I see you removed the patch. Did you forget to add the new version?

File: patches/app-builder-lib+22.7.0.patch [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+@@ -135,7 +135,7 @@ class LinuxTargetHelper {
+         exec += executableArgs.join(" ");
+       }
+ 
+-      exec += " %U";
++      exec += " --no-sandbox %U";
+     }
+ 
+     const desktopMeta = {
+diff --git a/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js.orig b/node_modules/app-builder-lib/out/targets/LinuxTargetHelper.js.orig
```
Comment (saghul) [inline patches/app-builder-lib+22.7.0.patch:14]: did you generate this by hand or let electron-patcher do it? This second part shouldn't be here I think.

**Comment by awlx** [comment]
Whoops indeed ... sorry. It's now included.

File: patches/app-builder-lib+22.7.0.patch (hunk 1 above)
Comment (awlx) [reply patches/app-builder-lib+22.7.0.patch:14]: Did this with:
npx patch-package app-builder-lib

Comment (awlx) [reply patches/app-builder-lib+22.7.0.patch:14]: Maybe we can just safely remove that part?

Comment (saghul) [reply patches/app-builder-lib+22.7.0.patch:14]: I think so, yeah. Can you please double-check though?

Comment (awlx) [reply patches/app-builder-lib+22.7.0.patch]: Verified with https://travis-ci.com/github/freifunkMUC/jitsi-meet-electron/jobs/342565173#L450

And committed

**Comment by saghul** [comment]
Thank you very much!

**Pull request merged**